src/frontend/src/lib/services/loader.services.ts

```diff
@@ -0,0 +1,37 @@
+import { allowSigning } from '$lib/api/backend.api';
+import { errorSignOut } from '$lib/services/auth.services';
+import { authStore } from '$lib/stores/auth.store';
+import { i18n } from '$lib/stores/i18n.store';
+import type { ResultSuccess } from '$lib/types/utils';
+import { get } from 'svelte/store';
+
+/**
+ * Initializes the signer allowance by calling `allow_signing`.
+ *
+ * This function should be called once during boot time before retrieving ETH or BTC addresses.
+ * It allocates a cycles budget sufficient for a reasonable number of signer calls.
+ * A "reasonable" number is currently defined as 30 calls, allowing the user to retrieve their ETH and BTC addresses
```

Comment:
I know this is not the scope of this PR, but is this number definitive? What was the reason behind 30? You most probably already discussed it ad-nauseam

Reply:
Comment is literally what @bitdivine told me. So is probably better suited to answer your question.

Reply:
I will ask separately, thank you!